tailwind.config.js

```diff
@@ -360,7 +362,24 @@ module.exports = {
     justifyContent: ['group-hover']
   },
   plugins: [
-    require('tailwindcss-plugins/gradients'),
+    plugin(function({ addUtilities, e, theme, variants }) {
+      const utilities = _.map(theme('gradients', {}), (gradient, name) => {
+        const type =
+          _.isPlainObject(gradient) && gradient.hasOwnProperty('type')
+            ? gradient.type
+            : 'linear';
+        const colors = _.isPlainObject(gradient)
+          ? gradient.colors || []
+          : gradient;
+
+        return {
+          [`.bg-${e(name)}`]: {
+            backgroundImage: `${type}-gradient(${colors.join(', ')})`
+          }
+        };
+      });
+      addUtilities(utilities, variants('gradients', []));
+    }),
```

Comment:
this is copied from a small package we imported thats now been abandoned and had clashing peer dependancies